lib/featureDetection.js

```diff
@@ -32,7 +32,7 @@ function getConfigVarFlag(stdout) {
 }
 
 function getPathConfigFlag(stdout) {
-  if (stdout.indexOf(' --path.conf ') > -1) {
+  if (stdout.indexOf('-Dproperty=value') > -1) {
```

Comment:
oh geez, that's what I wanted.  I was scanning the help text for --path.conf and ended up choosing the first difference I saw.  Thanks.